src/Features/LanguageServer/Protocol/Features/Options/MetadataAsSourceOptionsStorage.cs

```diff
@@ -18,7 +18,9 @@ public static MetadataAsSourceOptions GetMetadataAsSourceOptions(this IGlobalOpt
             NavigateToSourceLinkAndEmbeddedSources = globalOptions.GetOption(NavigateToSourceLinkAndEmbeddedSources),
         };
 
-    public static Option2<bool> NavigateToDecompiledSources = new("dotnet_navigate_to_decompiled_sources", defaultValue: true);
-    public static Option2<bool> AlwaysUseDefaultSymbolServers = new("dotnet_always_use_default_symbol_servers", defaultValue: true);
-    public static Option2<bool> NavigateToSourceLinkAndEmbeddedSources = new("dotnet_navigate_to_source_link_and_embedded_sources", defaultValue: true);
+    private static readonly OptionGroup s_goToDefinitionGroup = new(name: "go_to_definition", description: "");
```

Comment:
>go_to_definition [](http://example.com/codeflow?start=75&length=16)

How about `navigate`?

Reply:
😅The strange thing is in VS it would be called 'go to definition'
<img width="337" alt="image" src="https://user-images.githubusercontent.com/24360909/223824989-f691a1f0-9791-4960-8f54-7f9810e1d1dc.png">
And the option storage is called 'MetadataAsSource'
Then later the group is called 'navigation'

I am fine with all these names but it feels like a thing we need to finalize with the feature owner later.

Reply:
I will leave this comment open

Reply:
Go To Definition is too specific. I don't think we want to have Go To Definition, Go To Implementation, Go To Metadata, etc.
`navigation` is a more general term that encompasses all of these.

Reply:
I wouldn't worry about the VS UI.